create-snowpack-app/app-template-preact-typescript/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "include": ["src", "types"],
+  "exclude": ["node_modules"],
```

Comment:
I think ```"exclude": ["node_modules"]``` can be removed from all tsconfigs in all typescript templates

Reply:
Yeah good call. If you wanted to remove that as part of this PR, I’d be fine with that.